.github/workflows/Documenter.yml

```
@@ -45,3 +45,24 @@ jobs:
           DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }} # For authentication with SSH deploy key
           GKSwstype: 100 # To make GitHub Action work, disable showing a plot window with the GR backend of the Plots package
         run: julia --project=docs --color=yes docs/make.jl
+
+  build-doxygen:
+    runs-on: ubuntu-latest
+    needs: build-docs
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Install doxygen
+        run: |
+          sudo apt-get install -y doxygen
+      - name: Build doxygen docs
+        run: |
+          cd docs/doxygen
+          doxygen
+      - name: Deploy doxygen docs (only on main)
+        # if: ${{ github.ref == 'refs/heads/main' }} # TODO remove comment before merge!
```

```suggestion
        if: ${{ github.ref == 'refs/heads/main' }}
```
Needs to be applied before merging the PR, otherwise future PR builds will overwrite the main docs

---

Not needed anymore.